src/main/java/app/AppBuilder.java

```diff
@@ -20,10 +21,15 @@
 import interface_adapter.moon_phase.MoonPhasePresenter;
 import interface_adapter.star_chart.StarChartController;
 import interface_adapter.star_chart.StarChartPresenter;
+import interface_adapter.near_earth_objects.NEOController;
+import interface_adapter.near_earth_objects.NEOPresenter;
 import use_case.events.EventsInteractor;
 import use_case.moon_phase.MoonPhaseInteractor;
 import use_case.geocoding.GeocodingInteractor;
 import use_case.star_chart.StarChartInteractor;
+import use_case.near_earth_objects.NEOInteractor;
+import view.NEOView;
```

Comment:
Don't need this import or the import underneath (line 32) because ``import view.*`` on line 33 will import all classes in the view package